src/config.rs

```diff
@@ -612,7 +836,10 @@ struct DefaultPanicMessage;
 
 impl PanicMessage for DefaultPanicMessage {
     fn display(&self, pi: &std::panic::PanicInfo<'_>, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        writeln!(f, "{}", "The application panicked (crashed).".red())?;
+        // XXX is my assumption correct that this function is guaranteed to only run after `color_eyre` was setup successfully (including setting `STYLES`), and that therefore the following line will never panic? Otherwise, we could return `fmt::Error`, but if the above is true, I like `unwrap` + a comment why this never fails better
```

Comment:
I'll have to check that this doesn't happen elsewhere but my expectation is that if someone forgets to call the `install` fn then this could cause a double panic. Assuming that's the case I think we'd want to do a `get_or_init` and provide the default style incase it hasn't been setup elsewhere.

Reply:
My thinking was, that `get_or_init` could lead to confusion here (because suddenly the default theme is used) and that panicking would be better, therefore. `fmt::Error` wouldn't be ideal for the same reason (it's non-descriptive).

> I'll look into that later today to make sure this won't introduce any problems when they're both merged.

Alright. just let me know, what your conclusion was and I'll change this part.